parsers/NZ.py

```diff
@@ -109,6 +109,7 @@ def fetch_production(
             ],
             "wind": productions.get("Wind", {"generation": 0.0})["generation"],
             "hydro": productions.get("Hydro", {"generation": 0.0})["generation"],
+            "solar": productions.get("Solar", {"generation": 0.0})["generation"],
```

Comment:
It would probably be better to default to `None` rather than `0.0` while we don't know. 0.0 indicates that it's known to be 0. None indicates that we don't know.

Reply:
I've defaulted all to None, except Nuclear which for the foreseeable future will always be 0